apps/webservice/src/app/[workspaceSlug]/systems/[systemSlug]/deployments/[deploymentSlug]/releases/[versionId]/PolicyApprovalRow.tsx

```diff
@@ -42,28 +50,67 @@ export const PolicyApprovalRow: React.FC<PolicyApprovalRowProps> = ({
       router.refresh();
       utils.environment.policy.invalidate();
       utils.job.config.invalidate();
-      toast.success(`Approved release to ${environmentNames}`);
+      toast.success(`Approved release to ${environmentName}`);
     },
     onError: () => toast.error("Error approving release"),
   });
 
-  const environmentNames = environments.map((e) => e.name).join(", ");
-  const handleReject = () => rejectMutation.mutate({ releaseId, policyId });
-  const handleApprove = () => approveMutation.mutate(approval);
+  const handleReject = () =>
+    rejectMutation.mutate({
+      releaseId,
+      policyId,
+      userId: currentUserID!,
```

Comment:
```suggestion
      userId: currentUserId!,
```

look at the variables above....